security/pkg/nodeagent/caclient/client.go

```diff
@@ -54,6 +54,6 @@ func NewCAClient(endpoint, CAProviderName string, tlsFlag bool) (caClientInterfa
 	case googleCA:
 		return gca.NewGoogleCAClient(conn), nil
 	default:
-		return nil, nil
+		return nil, fmt.Errorf("CA provider %q isn't supported", CAProviderName)
```

Comment:
nit, maybe it's worth to output the supported list, for now it would be only "googleCA".

Reply:
done